<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vcsTool</class>
 <widget class="QMainWindow" name="vcsTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>vcsTool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QComboBox" name="secBox"/>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Sections</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="sectionsView">
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="insertSectionView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Features</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="featuresView">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Schematic</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="newAct">
   <property name="text">
    <string>New...</string>
   </property>
   <property name="toolTip">
    <string>Create new .db file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openAct">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open .db file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>secBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>vcsTool</receiver>
   <slot>setSec(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>346</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>347</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setSec(int)</slot>
 </slots>
</ui>
